go/kbcmf/packets.go

```diff
@@ -7,38 +7,42 @@ import ()
 
 type receiverKeysPlaintext struct {
 	_struct    bool   `codec:",toarray"`
-	GroupID    uint32 `codec:"gid"`
-	MACKey     []byte `codec:"mac,omitempty"`
-	SessionKey []byte `codec:"sess"`
+	Sender     []byte `codec:"sender"`
+	SessionKey []byte `codec:"session_key"`
 }
 
 type receiverKeysCiphertexts struct {
-	_struct bool   `codec:",toarray"`
-	KID     []byte `codec:"key_id"`
-	Keys    []byte `codec:"keys"`
-	Sender  []byte `codec:"sender"`
+	_struct     bool   `codec:",toarray"`
+	ReceiverKID []byte `codec:"receiver_key_id"`
```

Comment:
I test both cases, but not what the encoding is.  I'm pretty sure it's null, and not a zero-length array. I'll put a test in for a zero-length byte-array.  I'd want that to produce an error, no?